src/style.css

```diff
@@ -0,0 +1,33 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.list {
+  width: 96px;
+  height: 96px;
+}
+
+.stars__star {
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  list-style: none;
+  background-image: url(images/star.svg);
+}
+
+.stars {
+  padding: 0;
```

Comment:
delete all styles in selector except display: flex;
and add margin margin-right: 4px; to all stars except last child